### DISCUSSION TITLE
On mobile chrome, device model recognized as K

### DISCUSSION THREAD

**Discussion by PersonTest1** [faisalman/ua-parser-js#678] [General]
Using my Samsung phone, I browsed https://faisalman.github.io/ua-parser-js/#try using **chrome**. It can recognize the device model correctly. However when I implemented the js file on a website, it's always recognized as K. What could be the problem? When I tried using Samsung Browser, it can recognize the device well. The problem is always with Chrome browser. I tried to use another brand phone, Xiaomi and it's also recognized as K but if I use default browser from Xiaomi, it works well.

**Comment by faisalman** [comment]
The script in demo page is using `withClientHints()` method to improve the detection on Chrome browsers

https://docs.uaparser.js.org/v2/api/ua-parser-js/idata/with-client-hints.html

**Reply by NickSutton** [reply]
This link is down, can you signpost me to the detail on `withClientHints()` please?

**Reply by faisalman** [reply]
https://docs.uaparser.dev/api/ua-parser-js/idata/with-client-hints.html

**Comment by PersonTest1** [comment]
Thank you for the quick response. It works well
